source/samples/pubsub/PubSubFeature.cpp

```diff
@@ -69,6 +69,11 @@ bool PubSubFeature::createPubSub(const PlainConfig &config, std::string filePath
         {
             return false;
         }
+
+        if (payload != NULL)
+        {
```

Comment:
Add a comment over here since on line #67 we say we are creating an empty file. 

`Write payload data in newly created empty file.`